### PR TITLE
feat: show an input box to get identifier

### PR DESCRIPTION
**Pull request by xiaofengleo**
In AgentShape, the Identifier is mandatory. The publisher is of type AgentShape, it is required to input an ID for the publisher. A box should be displayed for a user to input the ID, that requeirs the dash:viewer dash:LiteralViewer ;  dash:editor dash:TextFieldEditor ; for the identifier.

### CODE REVIEW

File: Formalisation(shacl)/Core/PiecesShape/Dataset.ttl [hunk 1]
```diff
@@ -132,14 +132,23 @@
...
     sh:nodeKind sh:Literal ;
-    sh:path foaf:name ;
-  ] ;
-  sh:targetClass foaf:Agent .
+    dash:viewer dash:LiteralViewer ;
+    dash:editor dash:TextFieldEditor ;
+    sh:minCount 1 ;
+    sh:maxCount 1 ;
+  ];
+sh:targetClass foaf:Agent .
```
Comment (kburger) [inline Formalisation(shacl)/Core/PiecesShape/Dataset.ttl:154]: Missing newline at EOF.

File: Formalisation(shacl)/Core/PiecesShape/Catalog.ttl
Comment (kburger) [inline Formalisation(shacl)/Core/PiecesShape/Catalog.ttl:1]: Very inconsistent use of tabs/spaces for indentation.

**Review by kburger — CHANGES_REQUESTED** [review]

File: Formalisation(shacl)/Core/PiecesShape/Catalog.ttl
Comment (xiaofengleo) [reply Formalisation(shacl)/Core/PiecesShape/Catalog.ttl:1]: done

**Review by kburger — APPROVED** [review]
Looks good enough to me.

**Pull request merged**